pkgs/development/python-modules/jaraco_classes/default.nix

```diff
@@ -1,19 +1,32 @@
-{ buildPythonPackage, fetchPypi, isPy27, setuptools_scm, six, more-itertools }:
+{ lib, buildPythonPackage, fetchFromGitHub, isPy27
+, setuptools_scm, toml
+, more-itertools
+, pytestCheckHook
+}:
 
 buildPythonPackage rec {
   pname = "jaraco.classes";
   version = "3.1.1";
   disabled = isPy27;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "92bf5d4e6957b17f21034c956dc90977f8ef496c3919ccb165f457f0e2f63cac";
+  src = fetchFromGitHub {
+    owner = "jaraco";
+    repo = "jaraco.classes";
+    rev = "v${version}";
+    sha256 = "0wzrcsxi9gb65inayg0drm08iaw37jm1lqxhz3860i6pwjh503pr";
   };
 
-  pythonNamespaces = [ "jaraco" ];
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
```

Comment:
the `pythonNamespaces = [ "jaraco" ];` change should be added back to avoid pep420 conflicts
```
$ ls ./result/lib/python3.8/site-packages/jaraco/
__init__.py  __pycache__  classes
```